lib/cms/hippo_importer/base.rb

```diff
@@ -0,0 +1,99 @@
+module Cms
+  module HippoImporter
+    class Base
+      def initialize(data:, docs:, to: 'en', parser: HippoXmlParser, html_decoder: HTMLEntities.new)
+        @records = parser.parse(data, [hippo_type])
+        @_site = to || 'en'
+        @docs = docs || []
+        @html_decoder = html_decoder
+      end
+
+      attr_reader :records, :html_decoder
+
+      def site
+        @site ||= Comfy::Cms::Site.find_by(label: @_site)
+      end
+
+      def hippo_type
+        'contentauthoringwebsite:Guide'
+      end
+
+      def layout
+        @_layout ||= (site.try(:layouts) || []).first
```

Comment:
Acctually, the old code is wrong. All the site has layouts. This hippo importer should not have this. I will use just site.layouts.find_by(identifier: 'article') # because  this should be default. Thanks